packages/cli/src/templates/utils/modify-json-file.ts

```diff
@@ -0,0 +1,16 @@
+import { readFile, writeFile } from "node:fs/promises";
+
+export async function modifyJsonFile<T>(
```

Comment:
This is good - but to avoid type errors I would recommend defaulting the `T` to `Record<string, any>`


```suggestion
export async function modifyJsonFile<T = Record<string, any>>(
```

Reply:
ahhhh there it is! thanks